Not sure why the build-full-tests-2204 is failing.  The test failing is
```
_____________________________ test_missing_frames ______________________________
..
    def test_missing_frames(drop_frame_data):
    
        exp_vis = np.identity(4)[np.triu_indices(4)]
        frac_var = 1.0 / (
            drop_frame_params["samples_per_data_set"]
            * drop_frame_params["num_gpu_frames"]
            * (1 - drop_frame_params["drop_probability"])
        )
    
        vis_set = np.array([frame.vis for frame in drop_frame_data])
        weight_set = np.array([frame.weight for frame in drop_frame_data])
    
        # These tests need a 5 sigma fluctuation to cause failure
        assert np.allclose(vis_set.mean(axis=0), exp_vis, atol=7e-4, rtol=0)
>       assert np.allclose(vis_set.var(axis=0), frac_var, rtol=7e-2, atol=0)
E       assert False
E        +  where False = <function allclose at 0x7f79b47b50b0>(array([0.00014041, 0.00014264, 0.00014518, 0.00014519, 0.00014295,\n       0.00014545, 0.00014157, 0.00015504, 0.00014483, 0.00013705],\n      dtype=float32), 0.00014285714285714287, rtol=0.07, atol=0)
E        +    where <function allclose at 0x7f79b47b50b0> = np.allclose
E        +    and   array([0.00014041, 0.00014264, 0.00014518, 0.00014519, 0.00014295,\n       0.00014545, 0.00014157, 0.00015504, 0.00014483, 0.00013705],\n      dtype=float32) = <built-in method var of numpy.ndarray object at 0x7f79a6974e10>(axis=0)
E        +      where <built-in method var of numpy.ndarray object at 0x7f79a6974e10> = array([[ 1.000923  +0.j        ,  0.01246154-0.01138461j,\n        -0.00984615+0.00769231j, ...,  1.0043076 +0.j        ,\n         0.00507692-0.00615385j,  0.99153847+0.j        ],\n       [ 0.9991428 +0.j        , -0.00714286-0.001j     ,\n        -0.00542857+0.00014286j, ...,  0.99957144+0.j        ,\n        -0.00314286+0.00171429j,  1.006     +0.j        ],\n       [ 0.9855882 +0.j        , -0.01764706+0.00441176j,\n         0.01323529-0.0025j    , ...,  1.000147  +0.j        ,\n        -0.00279412+0.00514706j,  0.9913235 +0.j        ],\n       ...,\n       [ 1.0017188 +0.j        ,  0.025     +0.0128125j ,\n        -0.01140625+0.j        , ...,  1.0010937 +0.j        ,\n        -0.01078125+0.0003125j ,  1.0098437 +0.j        ],\n       [ 1.0115873 +0.j        ,  0.00571429-0.00380952j,\n        -0.0115873 +0.00571429j, ...,  1.0039682 +0.j        ,\n         0.00666667+0.0047619j ,  1.0033333 +0.j        ],\n       [ 1.01      +0.j        ,  0.00294872-0.02179487j,\n         0.00487179-0.00871795j, ...,  0.99512815+0.j        ,\n        -0.00333333+0.00717949j,  1.0047436 +0.j        ]],\n      dtype=complex64).var

test_accumulate.py:373: AssertionError
```

--- a/lib/core/buffer.cpp
+++ b/lib/core/buffer.cpp
@@ -981,6 +981,7 @@ uint8_t* buffer_malloc(size_t len, int numa_node, bool use_hugepages, bool mlock
 void buffer_free(uint8_t* frame_pointer, size_t size, bool use_hugepages) {
 #ifdef WITH_HSA
     (void)size;
+    (void)use_hugepages;
     hsa_host_free(frame_pointer);
 #else
     if (use_hugepages) {

--- a/lib/core/buffer.cpp
+++ b/lib/core/buffer.cpp
@@ -909,7 +909,8 @@ uint8_t* buffer_malloc(size_t len, int numa_node, bool use_hugepages, bool mlock
     uint8_t* frame = NULL;
 
 #ifdef WITH_HSA // Support for legacy HSA support used in CHIME
-    frame = hsa_host_malloc(len, numa_node);
+    (void)use_hugepages;
+    frame = (uint8_t*)hsa_host_malloc(len, numa_node);
     if (frame == NULL) {
         return NULL;
     }

